test/integration/cli/test/index.test.js

```diff
@@ -693,23 +693,13 @@ describe('CLI Usage', () => {
       expect(stderr).not.toContain('UnhandledPromiseRejectionWarning')
     })
 
-    test('should exit when SIGINT is signalled', async () => {
+    test.only('should exit when SIGINT is signalled', async () => {
       const port = await findPort()
-      await testExitSignal(
-        'SIGINT',
-        ['dev', dirBasic, '-p', port],
-        /- Local:/,
-        'SIGINT'
-      )
+      await testExitSignal('SIGINT', ['dev', dirBasic, '-p', port], /- Local:/)
```

Comment:
Since these signals are now handled by us and the process is exited by us, this is no longer detectable.